src/autocomplete/autocomplete-types.ts

```diff
@@ -45,6 +45,26 @@ export interface ClickHouseAutocompleteResult extends AutocompleteResultBase {
     suggestEngines?: EngineSuggestion;
 }
 
+type YQLEntity =
+    | 'externalDataSource'
+    | 'view'
+    | 'object'
+    | 'tableStore'
+    | 'table'
+    | 'replication'
+    | 'topic'
+    | 'group'
+    | 'user';
```

Comment:
You need to export this field, because your end users might want to use it in their code. Also please add a newline between `YqlEntity` and `YQLAutocompleteResult`